app/src/main/java/com/example/farmeraid/uicomponents/BottomNavigationBar.kt

```diff
@@ -46,6 +46,7 @@ fun BottomNavigationBar(appNavigator: AppNavigator) {
     )
     val notShownScreens: List<String> = listOf(
         NavRoute.SignIn.route,
+        NavRoute.SignUp.route
```

Comment:
Nice catch adding this in here. Forgot to mention that you need to do this.

Reply:
Ye no worries, don't want nav bar to show on signup XD